src/init.cpp

```diff
@@ -135,7 +157,11 @@ void updateConfigurationValue(const pika::program_options::variables_map& vm, T&
   const std::string dlaf_env_var = "DLAF_" + env_var;
   char* env_var_value = std::getenv(dlaf_env_var.c_str());
   if (env_var_value) {
-    var = parseFromString<T>::call(env_var_value);
+    if (auto parsed_value = parseFromString<T>::call(env_var_value))
+      var = parsed_value.value();
+    else
+      std::cerr << "Environment variable " << dlaf_env_var << " has an invalid value (='"
+                << env_var_value << "').\n";
```

Comment:
How about we terminate instead? It's what I presume would happen soon after if e.g. `parseFromString<std::size_t>` throws because of an invalid value. It's the fastest way to get users(/us) to fix wrongly set environment variables instead of quietly continuing.

Reply:
see https://github.com/eth-cscs/DLA-Future/pull/913/commits/589e5ac478b63ae1eaaf313552fc598d4016eeb2

Reply:
Thanks @albestro, looks good to me. In the future we might also want to say what are actually valid values, instead of just saying that it's invalid, but let's not overcomplicate things now.